### PR TITLE
historyScreen and historyDetailScreen display query values

### PR DESCRIPTION
**Pull request by oozywaters**

### CODE REVIEW

**Comment by oozywaters** [comment]
Ready for review.

File: react/e2e/private/historyScreen/inventory.test.js [hunk 1]
```diff
@@ -14,4 +14,12 @@ describe('historyScreen inventory', () => {
     )
     expect(currentAccountBalanceIndicator).toEqual(1)
   })
+
+  it('displays 20 most recent historyItemIndicator in descending time', async () => {
```
Comment (mxfactorial) [inline react/e2e/private/historyScreen/inventory.test.js]: "20" and "most recent" not tested here

please change title to describe current test

**Comment by mxfactorial** [comment]
brief redemptions:
1. `disputeTransactionButton` displays a [graphic](https://github.com/systemaccounting/mxfactorial/blob/0c24f2db8c7f46d3b898dab68d468611f1750023/react/src/screens/HistoryDetailScreen/HistoryDetailScreen.js#L157) from a [globally imported library](https://github.com/systemaccounting/mxfactorial/blob/0c24f2db8c7f46d3b898dab68d468611f1750023/react/src/App.js#L3) instead of [asset](https://drive.google.com/file/d/14FyoVE_4h7JNYRvuj_PSAMTblPsKziUF/view) required in [sequence](https://docs.google.com/drawings/d/1W4KTDjrqXGz7Xvv56529XBKnOoDdwBZzqUyHPuYQGbE/edit?usp=sharing). please switch to required asset by adding icon to [react/src/icons](https://github.com/systemaccounting/mxfactorial/tree/7ceeea63f56290c06b4235e1a59241cd5edab78d/react/src/icons) then displaying

    icon color configurable in .svg's `fill` attribute of `g` element

2. please remove [logger](https://github.com/systemaccounting/mxfactorial/blob/c595e0b2c4d226a795e45244f8338fa096e788e6/react/src/screens/HistoryDetailScreen/HistoryDetailScreen.js#L40)

**Comment by oozywaters** [comment]
Ready for review.

**Pull request merged**